### PR TITLE
Components part 1

### PR DESCRIPTION
**Pull request by olgaDolinger**
Components part 1

### CODE REVIEW

File: webpack.config.prod.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    path: path.join(__dirname, "build"),
+    filename: "bundle.js",
+  },
+
+  resolve: {
+    extensions: [".js", ".jsx", ".json"],
+    modules: [path.resolve(__dirname, "./src"), "./node_modules"],
+  },
+
+  devServer: {
```
Comment (lotrien) [inline webpack.config.prod.js:22]: Why do we need dev server option for prod config? How is it going to help?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    "prop-types": "^15.7.2",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-router": "^5.2.0",
+    "redux": "^4.0.5",
+    "styled-components": "^5.2.1",
+    "ts-loader": "^8.0.17",
+    "typescript": "^4.2.2"
+  }
+}
```
Comment (nataliakonovalenko) [inline package.json:63]: In this case, there is no need to use babel/prop-types & typescript. Please choose one option.

File: src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import Header from "./header/Header";
+import MainPage from "./main-page/MainPage";
+import Footer from "./footer/Footer";
+import * as S from "./App.styled";
+
+const App = () => {
+  return (
+    <>
```
Comment (nataliakonovalenko) [inline src/App.tsx:9]: Fragments let you group a list of children without adding extra nodes to the DOM.
Why do you use a fragment if there is one common <div> in the component?

File: webpack.config.prod.js (hunk 1 above)
Comment (olgaDolinger) [reply webpack.config.prod.js:22]: thanks, devserver removed

File: src/App.tsx (hunk 3 above)
Comment (olgaDolinger) [reply src/App.tsx]: thanks, fragment was removed

File: package.json (hunk 2 above)
Comment (olgaDolinger) [reply package.json:62]: I removed prop-types, thanks

Comment (nataliakonovalenko) [reply package.json:62]: Hi Olga,
In current project, you used extension .tsx , but didn't use all possibilities of typescript I mean writing an interface to check data types.

The quick solution for this case is to change all .tsx files to .jsx and return prop-types.

**Pull request merged**